### DISCUSSION TITLE
Questions about solving the rectangle in front of the notes in the file panel?

### DISCUSSION THREAD

**Discussion by Woohwah** [BryanJenksCommunity/FAQ#164] [Q&A] [answered]
Hi, I just imported your latest template and found that there seems to be some bugs in the file panel. There will be an undeletable rectangle in front of the name of each file or folder (except the vault itself) I'm using the windows platform OBSIDIAN version 1.0.3. Maybe the problem is in the CSS of the theme?
![image](https://user-images.githubusercontent.com/80728302/207578496-684226a1-7dad-4403-932d-84c3377ee6fa.png)

**Comment by Woohwah** [comment, marked as answer]
Never mind. I just figured it out.
https://github.com/kmaasrud/awesome-obsidian#resources

**Reply by spandankeshav** [reply]
Dude how did you manage that?

**Reply by tallguyjenks** [reply]
the more helpful link is this: https://github.com/kmaasrud/awesome-obsidian/blob/master/code/css-snippets/custom-icons-differing-files-and-folders.css
